File: samples/python/number_format.py
"""
Average of two numbers.  Bug: the float is printed without formatting.
autotester should say: number formatting, print with 2 decimal places.

Example execution 1:
Enter the first number: -> 3
Enter the second number: -> 4
The average is 3.50

Example execution 2:
Enter the first number: -> 10
Enter the second number: -> 5
The average is 7.50

Example execution 3:
Enter the first number: -> 2
Enter the second number: -> 2
The average is 2.00
"""
a = int(input("Enter the first number: "))
b = int(input("Enter the second number: "))
print(f"The average is {(a + b) / 2}")

File: samples/other-languages/number_format.c
/*
Average of two numbers.  Bug: printf uses %.1f where the example shows 2 decimals.
autotester should say: number formatting, and name the printf line.

Example execution 1:
Enter the first number: -> 3
Enter the second number: -> 4
The average is 3.50

Example execution 2:
Enter the first number: -> 10
Enter the second number: -> 5
The average is 7.50
*/
#include <stdio.h>
int main(void) {
    double a, b;
    printf("Enter the first number: ");
    scanf("%lf", &a);
    printf("Enter the second number: ");
    scanf("%lf", &b);
    printf("The average is %.1f\n", (a + b) / 2);
    return 0;
}
